bitnami/minio/templates/deployment-standalone.yaml

```
@@ -74,19 +74,19 @@ spec:
               value: {{ ternary "true" "false" .Values.image.debug | quote }}
             - name: MINIO_FORCE_NEW_KEYS
               value: {{ ternary "yes" "no" .Values.forceNewKeys | quote }}
-            {{- if .Values.useCredentialsFile }}
+            {{- if .Values.credentialsFile.enabled }}
             - name: MINIO_ACCESS_KEY_FILE
-              value: "/opt/bitnami/minio/secrets/access-key"
+              value: {{ .Values.credentialsFile.mountPath }}{{ .Values.credentialsFile.accessKeyFile }}
```

For safety, I would use a `/` between template expansions, in camse mountPath is provided without the final slash.

```suggestion
              value: {{ .Values.credentialsFile.mountPath }}/{{ .Values.credentialsFile.accessKeyFile }}
```

Please update the rest of paths, if you think this suggestion is useful.

---

Yup, sounds good. I updated all the paths with a dividing /